### PR TITLE
 Add integration version subcommand to get the installed version of a check

### PR DESCRIPTION
**Pull request by zippolyte**
### What does this PR do?

Print currently installed version of the specified package

```
$ datadog-agent integration show datadog-vsphere
Package datadog-vsphere:
Installed version: 3.6.0

$ datadog-agent integration show datadog-vsphere -q
3.6.0
```

Exit with 0 and prints nothing if the package is not installed 

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: cmd/agent/app/integrations.go [hunk 1]
```diff
@@ -608,3 +620,33 @@ func freeze(cmd *cobra.Command, args []string) error {
...
+	if version == nil {
+		// Package not installed, return 0 and print nothing
+		return nil
+	}
+
+	if quietVersion {
+		// Print only the version for easier parsing
+		fmt.Println(version)
+	} else {
+		fmt.Printf("Installed version of %s: %s", packageName, version)
```
Comment (hush-hush) [inline cmd/agent/app/integrations.go]: You could just add a `\n` at the end of the first argument and remove the next line.

**Review by hush-hush — DISMISSED** [review]

**Review by olivielpeau — COMMENTED** [review]
This looks good, but actually I've changed my mind on what the command's interface should be 😛 I feel the command would make more sense if it was:

* called `show` instead of `version` (I feel `version` is generally meant to only display the version of the tool being used, not the version of what it installs)
* returned more info on the integration (as a first approach I guess the name and version are enough)
* had a `--show-version-only` flag to only display the version of the integration

What do you think?

**Review by olivielpeau — DISMISSED** [review]
💯

**Review by ofek — DISMISSED** [review]
Thanks!

**Review by olivielpeau — APPROVED** [review]
:shipit:

**Pull request merged**